src/librustdoc/html/static/main.js

```diff
@@ -2615,6 +2652,10 @@ function getSearchElement() {
         insertAfter(popup, getSearchElement());
     }
 
+    handleHashes();
+    highlightSourceLines();
```

Comment:
Would it be more future-proof to just call `onHashChange()` here?

Reply:
Took me a while to think about it but I think you're right. I updated the code. Thanks!